core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -672,164 +658,605 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
     </script>
   }
 
-  def taskRow(
```

Comment:
This method is split to two methods: `TaskDataSource.taskRow` and `TaskPagedTable.row`.